tasks/test.rake

```diff
@@ -17,6 +17,12 @@ namespace 'test' do
     t.cucumber_opts = 'features --format progress --color'
   end
 
+  task 'enable_simplecov' do
+    ENV['SIMPLECOV'] = '1'
+  end
+
+  task 'spec' => 'test:enable_simplecov'
```

Comment:
>> We don't want to enable it when running one spec file.

Obvious reason which I completely missed :laughing: 

>> Another option is probably to add -rsimplecov inside the spec task, so rspec something_spec.rb won't activate it.

Ah, I think that's a better solution, let's go for that!